dataproduct_apps/collect.py

```diff
@@ -113,4 +126,9 @@ def parse_apps(collection_time, cluster, apps, topics):
         app.read_topics.sort()
         app.write_topics.sort()
 
+        db_strings = []
+        for db in databases:
+            db_strings.append(db.resourceID + "." + db.databaseVersion + "." + db.tier)
```

Comment:
```suggestion
            db_strings.append(".".join((db.resourceID, db.databaseVersion, db.tier))
```

Reply:
Eller enda bedre, siden `Database` definerer `__str__`:

```suggestion
            db_strings.append(str(db))
```